presto-manager-agent/pom.xml

```diff
@@ -53,6 +53,54 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>com.github.kongchen</groupId>
+                <artifactId>swagger-maven-plugin</artifactId>
+                <version>3.1.5</version>
+                <configuration>
+                    <apiSources>
+                        <apiSource>
+                            <info>
+                                <title>API for Presto Manager Agent</title>
+                                <version>v1</version>
+                                <description>Swagger specification for REST API</description>
+                                <license>
+                                    <name>Apache License 2.0</name>
+                                    <url>https://github.com/prestodb/presto-manager/blob/master/LICENSE</url>
```

Comment:
Use the same URL as in the `licenses` tag. Then it will work for any forks, even if the repo changes or moves on GitHub.

`http://www.apache.org/licenses/LICENSE-2.0`

Because that URL is used so much, it might be a good idea to make it a variable. (But not the license name, because we can't change the project's license.)